doc/source/user/plugins.rst

```diff
@@ -335,6 +335,45 @@ As with other plugins, :class:`~watts.PluginABCE` is easily used by::
 .. note::
     `ABCE` is still under active development.
 
+A-LEAF Plugin
++++++++++++++
+The :class:`~watts.PluginALEAF` class enables simulations with the Argonne 
+Low-carbon Electricity Analysis Framework (A-LEAF) code using a text based 
+input file for fuel price modification. The A-LEAF code using an excel based
+input file, and the A-LEAF plugin will only modify the fuel price under the
+``Fuel`` section of the input file. Since A-LEAF code is still under
+active development, the plugin will be updated as the code is updated.
+
+The A-LEAF fuel price input file can be templated as follows:
+
+.. code-block:: jinja
+
+    Scenario	FUEL	Type	Unit	2022	2023	2024	2025	2026	...
+
+    Reference	Coal	Real	2022 $/MMBtu	1.64	1.61	1.65	1.57	1.52	...
+    ...
+    Base	Nuclear	Real	2020 $/MMBTU	{% for year, price in fuel_price.items() %}
+    {{ price }}{% if not loop.last %}	{% endif %}{% endfor %}
+    Base	Biomass	Real	2020 $/MMBTU	5.00	5.11	5.22	5.34	5.45	...
+    ...
+    Low	OGS	Real	2020 $/MMBTU	3.30	3.00	2.73	2.58	2.56	...
+        
+Before running the A-LEAF plugin, you need to specify the directory that the
+executable and the license key are in (they must be in the same directory). This
+can be done by adding the ``A-LEAF_DIR`` variable to the environment or by
+explicitly specifying the path in the Python script as::
+
+    aleaf_plugin = watts.PluginALEAF(
+        'aleaf_template',
+        executable="/path/to/A-LEAF",
+        extra_inputs=[]
+    )
```

Comment:
This doesn't seem to match the arguments available on the class (I don't see an `executable` argument)